modules/multi-arch-installing-using-cli.adoc

```diff
@@ -0,0 +1,120 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-installing-using-cli_{context}"]
+
+= Installing the Multiarch Tuning Operator by using the CLI
+
+You can install the Multiarch Tuning Operator by using the {oc-first}.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Log in to the {oc-first}.
+
+. Create a new project named `openshift-multiarch-tuning-operator` by running the following command:
++
+[source,terminal]
+----
+$ oc new-project openshift-multiarch-tuning-operator
+----
+
+. Create an `OperatorGroup` object:
```

Comment:
https://github.com/openshift/openshift-docs/pull/76628/files#diff-c492c9fcf48ac4ba2fbe83583c63d2fce4fbd53b644f55ebf2bffb5faf76d024R43 should be here (same as previous comments on the structure of these commands and example YAMLs)

Reply:
Please see the comments below.